sdk/dapp-kit/src/hooks/wallet/useSignPersonalMessage.ts

```diff
@@ -16,12 +16,19 @@ import { useCurrentAccount } from './useCurrentAccount.js';
 import { useCurrentWallet } from './useCurrentWallet.js';
 
 type UseSignPersonalMessageArgs = PartialBy<SuiSignPersonalMessageInput, 'account'>;
+
 type UseSignPersonalMessageResult = SuiSignPersonalMessageOutput;
 
+type UseSignPersonalMessageError =
```

Comment:
this is just some type tidying for errors - having strong types for possible error states is always nice. Technically someone could implement a bogus Sui feature that throws something other than an error... but that seems like a rare edge case that isn't worth handling.